raiden/tests/unit/transfer/mediated_transfer/test_initiator.py

```diff
@@ -127,9 +130,9 @@ def test_next_route():
     assert state.route == routes[0], 'a initialized state must use the first valid route'
 
     assert state.routes.available_routes == routes[1:]
-    assert len(state.routes.ignored_routes) == 0
-    assert len(state.routes.refunded_routes) == 0
-    assert len(state.routes.canceled_routes) == 0
+    assert not state.routes.ignored_routes
```

Comment:
I still find the `len(collection) == 0` more readable and explicit but it seems the pythonic way to do it is with the `not collection` so I am fine.